runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerConfig.java

```diff
@@ -45,17 +46,31 @@ default String getIgnoredItems()
 	@ConfigItem(
 		keyName = "ignoredItems",
 		name = "",
-		description = ""
+		description = "",
+		position = 2
 	)
 	void setIgnoredItems(String key);
 
+	@ConfigItem(
+		keyName = "valueType",
+		name = "Price display",
+		description = "Select which source is used to calculate loot prices",
+		position = 3
+	)
+	default LootTrackerValueType valueType()
+	{
+		return LootTrackerValueType.GRAND_EXCHANGE;
+	}
+
 	@ConfigItem(
 		keyName = "saveLoot",
 		name = "Save loot",
-		description = "Save loot between client sessions (requires being logged in)"
+		description = "Save loot between client sessions (requires being logged in)",
+		position = 4
 	)
 	default boolean saveLoot()
 	{
 		return true;
 	}
-}
\ No newline at end of file
+
```

Comment:
remove this empty line

Reply:
de0366819507da1b1d7aa15cc39f99176f5f9c62